### PR TITLE
feat: hide less used functions in overflow dropdown

### PR DESCRIPTION
**Pull request by dcshzj**
## Problem

Our Tiptap editor menu bar is a little crowded.

Also, this is preparation for adding the divider component back into Tiptap.

## Solution

**Breaking Changes**

- [ ] Yes - this PR contains breaking changes
- [x] No - this PR is backwards compatible

**Features**:

- Hide some of the less commonly used functions inside a dropdown menu. Currently, this is the superscript and subscript functions.

## Before & After Screenshots

**BEFORE**:

image

**AFTER**:

image

### CODE REVIEW

File: apps/studio/public/assets/css/preview-tw.css [hunk 1]
```diff
@@ -1766,6 +1766,10 @@ video {
   flex-shrink: 0;
 }
 
+.flex-grow {
+  flex-grow: 1;
+}
+
```
Comment (dcshzj) [inline apps/studio/public/assets/css/preview-tw.css:1772]: note: unrelated change to this PR, believe this came from #855.

**Review by adriangohjw — APPROVED** [review]
